src/types/point.js

```diff
@@ -6,7 +6,7 @@ export default class PointAnnotation extends Element {
 
   inRange(mouseX, mouseY, useFinalPosition) {
     const {width} = this.getProps(['width'], useFinalPosition);
-    return inPointRange({x: mouseX, y: mouseY}, this.getCenterPoint(useFinalPosition), width / 2 + this.options.borderWidth);
+    return inPointRange({x: mouseX, y: mouseY}, this.getCenterPoint(useFinalPosition), width / 2, this.options.borderWidth);
```

Comment:
I think this one already worked as expected

Reply:
the use case is the radius is 0 and `borderWidth` > 0. the `isPointRange` could return true with the previous implementation, instead, with radius = 0 should return false.